master/internal/config/config.go

```diff
@@ -42,6 +42,15 @@ const (
 	preemptionScheduler       = "preemption"
 )
 
+const (
+	// RSAKeyType uses RSA.
+	RSAKeyType = "RSA"
+	// ECDSAKeyType uses ECDSA.
+	ECDSAKeyType = "ECDSA"
+	// ED25519KeyType uses ED25519.
+	ED25519KeyType = "ED25519"
```

Comment:
Idiomatically, these names should be `KeyTypeRSA`, `KeyTypeECDSA`, and `KeyTypeED25519`. A good [example of this in the stdlib](https://pkg.go.dev/net/http#pkg-constants) is `net/http`, where all the methods and statuses are named this way.